src/Extension/Phiremock.php

```diff
@@ -75,7 +75,7 @@ public function startProcess()
         $this->process->start(
             $ip,
             $port,
-            realpath($this->config['bin_path']),
+            realpath(Config::projectDir() . $this->config['bin_path']),
```

Comment:
This will cause the configuration to support only relative paths. Maybe we could detect if it is relative or absolute and use the projectDIr method accordingly?